packages/common/components/Hero/index.js

```diff
@@ -33,7 +30,7 @@ const Hero = ({ title, button, onButtonClick }) => {
 Hero.propTypes = {
   title: PropTypes.string,
   button: PropTypes.string,
-  onButtonClick: PropTypes.func
+  onButtonClick: PropTypes.func,
```

Comment:
Something just come to my mind:
- Why we are not using TS or at least `flow`?

Reply:
This code is from the academics letter, I don't see the need to integrate flow for this since it's really basic, but for a new codebase I think we should go with TypeScript as flow configuration sometimes gives me problems